packages/terra-popup/src/terra-dev-site/doc/example/PopupDimensions.jsx

```diff
@@ -1,101 +1,97 @@
-    }
+  const contentDimensions = {};
+  if (popupContentHeight !== 'Default') {
+    contentDimensions.contentHeight = popupContentHeight;
+  }
 
-    if (this.state.popupContentWidth !== 'Default') {
-      contentDimensions.contentWidth = this.state.popupContentWidth;
-    }
+  if (popupContentWidth !== 'Default') {
+    contentDimensions.contentWidth = popupContentWidth;
+  }
 
-    const contentStyle = {};
-    if (this.state.popupContentHeight === 'auto') {
-      contentStyle.height = '500px';
-    }
-    if (this.state.popupContentWidth === 'auto') {
-      contentStyle.width = '500px';
-    }
-    return (
+  const contentStyle = {};
+  if (popupContentHeight === 'auto') {
+    contentStyle.height = '500px';
+  }
+  if (popupContentWidth === 'auto') {
+    contentStyle.width = '500px';
+  }
+  return (
+    <div>
+      <label htmlFor="popupContentHeight">Pop Content Height</label>
+      <select id="popupContentHeight" name="popupContentHeight" value={popupContentHeight} onChange={handleSelectHeightChange}>
+        <option value="Default">Default</option>
+        {generateOptions(HEIGHT_KEYS)}
+      </select>
+      <br />
+      <br />
+      <label htmlFor="popupContentWidth">Pop Content Width</label>
+      <select id="popupContentWidth" name="popupContentWidth" value={popupContentWidth} onChange={handleSelectWidthChange}>
+        <option value="Default">Default</option>
+        {generateOptions(WIDTH_KEYS)}
+      </select>
+      <br />
+      <br />
       <div>
-        <label htmlFor="popupContentHeight">Pop Content Height</label>
-        <select id="popupContentHeight" name="popupContentHeight" value={this.state.popupContentHeight} onChange={this.handleSelectChange}>
-          <option value="Default">Default</option>
-          {PopupDimensions.generateOptions(HEIGHT_KEYS)}
-        </select>
-        <br />
-        <br />
-        <label htmlFor="popupContentWidth">Pop Content Width</label>
-        <select id="popupContentWidth" name="popupContentWidth" value={this.state.popupContentWidth} onChange={this.handleSelectChange}>
-          <option value="Default">Default</option>
-          {PopupDimensions.generateOptions(WIDTH_KEYS)}
-        </select>
-        <br />
-        <br />
-        <div>
-          <Popup
-            {...contentDimensions}
-            classNameArrow="test-arrow"
-            classNameContent="test-content"
-            isOpen={this.state.open}
-            onRequestClose={this.handleRequestClose}
-            targetRef={() => document.getElementById('popup-dimensions')}
-          >
-            <PopupContent contentStyle={contentStyle} />
-          </Popup>
-          <Button id="popup-dimensions" text={`${this.state.popupContentHeight || 'Default'} x ${this.state.popupContentWidth || 'Default'} Popup`} onClick={this.handleButtonClick} />
-        </div>
+        <Popup
+          {...contentDimensions}
+          classNameArrow="test-arrow"
```

Comment:
Shouldn't need classNameArrow or classNameContent in the example page.

Reply:
Fixed here https://github.com/cerner/terra-framework/pull/927/commits/f26bebb7c61ac4b5747385bd2a04d26a4785fb3d.